src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -139,6 +140,23 @@ function SidebarLinksData({
     // eslint-disable-next-line react-hooks/exhaustive-deps
     useEffect(() => Policy.openWorkspace(activeWorkspaceID, policyMemberAccountIDs), [activeWorkspaceID]);
 
+    const reportsWithErrorsIds = useMemo(() => {
+        const reportKeys = _.keys(chatReports);
+        return _.reduce(
```

Comment:
Just a though, having 2 reduces, one nested the other, won't make this function O(n^2). 
can we create first the all reports.

```
const allReportsActions = {};
reportKeys.forEach((reportKey) => {
              const allReportsActionsMock = _.reduce(allReportActions[reportKey.replace('report_', 'reportActions_')], (acc, reportAction) => ({...acc, [reportAction.reportActionID]: reportAction}), {});
              allReportsActions = {...allReportsActionsMock, ...allReportsActionsMock}
});
```
and then later in the reduce you act as normally?? not sure if the code will work just guessing and code it here on the fly :)